playground/infrastructure/test_logger.py

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from logger import setup_logger
+
+
+def test_setup_logger():
+  count_default_handlers = len(logging.getLogger().handlers)
+  setup_logger()
+  assert (len(logging.getLogger().handlers) - count_default_handlers) == 2
```

Comment:
Why do we have 2 logger handlers? Could you please add a comment here why we expect it to equal 2

Reply:
@pavel-avilov please add a comment here